CoreWhatsNewView/src/au/gov/asd/tac/constellation/views/whatsnew/whatsnew.txt

```diff
@@ -1,10 +1,29 @@
 == 3030-12-31 Getting Started
 <p>If you're new to Constellation, read the <a href="" helpId="au.gov.asd.tac.constellation.functionality.gettingstarted">getting started guide</a>.</p>
 
+== 2023-08-08 - Enabled of Multi-Choice Drop-down bulk selection feature
+<p>A number of input field drop-downs that enable the selection of multiple items have had a bulk selection feature enabled.</p>
+<p>Right click the input field to access the "Select All" and "Clear All" functionality.</p>
+<p>Some instances have an adjacent drop-down labeled "..." that can be used to access the same functionality as right clicking.</p>
+
+== 2023-07-28 Attribute Editor View - Edit Buttons
+<p>Edit buttons have been removed from attributes in the Attributes View.</p>
+<p>Edit functionality can be achieved by clicking on the field of the attribute.</p>
+
+== 2023-07-28 Replacement of old Find View with new Find View
+<p>The existing Find View accessible using the shortcut Ctrl-F has been replaced with the an updated Find View which has been under development and testing.</p> 
+<p>The new Find View offers an updated interface that opens by default in a separate window.</p> 
+<p>The functionality remains the same with the added option of "Find Next" and "Find Previous" execution options.</p> 
+
 == 2023-06-23 Markdown support in the Notes View
```

Comment:
thinking it might be worth updating the date here and moving this up (given all the entries above relate to things that have already been merged in)